src/app/tests/BUILD.gn

```diff
@@ -81,13 +81,13 @@ source_set("ota-requestor-test-srcs") {
 
 source_set("scenes-table-test-srcs") {
   sources = [
-    "${chip_root}/src/app/clusters/scenes/ExtensionFieldSets.h",
-    "${chip_root}/src/app/clusters/scenes/ExtensionFieldSetsImpl.cpp",
-    "${chip_root}/src/app/clusters/scenes/ExtensionFieldSetsImpl.h",
-    "${chip_root}/src/app/clusters/scenes/SceneTable.h",
-    "${chip_root}/src/app/clusters/scenes/SceneTableImpl.cpp",
-    "${chip_root}/src/app/clusters/scenes/SceneTableImpl.h",
-    "${chip_root}/src/app/clusters/scenes/scenes-tokens.h",
+    "${chip_root}/src/app/clusters/scenes-server/ExtensionFieldSets.h",
```

Comment:
wouldn't `include_dirs = [ "${chip_root}/src/app/clusters/scenes-server" ]` here instead of listening all the headers

Reply:
Unless mistake on my part, including headers in sources has the benefit of triggering a rebuild upon modification in the header files only.

Reply:
meh fair enough